src/components/pages/home/latest-l1-l2-transactions/latest-l1-l2-transaction.tsx

```diff
@@ -0,0 +1,52 @@
+import AddressLink from "@/components/pages/home/AddressLink";
+import { SquareArrowOutUpRight } from "lucide-react";
+import { L1L2Transaction } from "@/lib/types";
+import { ReceiptText } from "lucide-react";
+import { l1Chain } from "@/lib/chains";
+
+interface LatestL1L2TransactionProps {
+  transaction: L1L2Transaction;
+}
+
+const LatestL1L2Transaction = ({ transaction }: LatestL1L2TransactionProps) => (
+  <div className="flex items-center gap-4 pt-6">
+    <div className="hidden h-9 w-9 items-center justify-center rounded-lg bg-accent text-accent-foreground 2xl:flex">
+      <ReceiptText className="h-5 w-5" />
+    </div>
+    <div className="grid gap-1">
+      <div className="flex items-center gap-1 text-sm">
+        Block#{" "}
+        <span className="flex max-w-60 items-center truncate text-primary md:max-w-96 xl:max-w-60">
```

Comment:
You removed the `a` anchor link here, making the external L1 block unclickable.
Double check you're not introducing regression when submitting PRs.